packages/flutter_tools/lib/src/test/flutter_tester_device.dart

```diff
@@ -261,6 +268,26 @@ class FlutterTesterTestDevice extends TestDevice {
     );
   }
 
+  Future<void> _startDevTools(Uri forwardingUri, DartDevelopmentService? dds) async {
+    _devToolsLauncher = DevtoolsLauncher.instance;
+    logger.printTrace('test $id: Serving DevTools...');
+    final DevToolsServerAddress? devToolsServerAddress = await _devToolsLauncher?.serve();
+    await _devToolsLauncher?.ready;
+    logger.printTrace('test $id: DevTools is being served at ${devToolsServerAddress?.uri}');
```

Comment:
do we still want to print this if it is null? Perhaps we should check `_devToolsLauncher != null` as the second line in this method, and only proceed if it is not null.